WORKSPACE

```diff
@@ -3,6 +3,19 @@ workspace(name = "pytorch")
 load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
 load("//tools/rules:workspace.bzl", "new_patched_local_repository")
 
+http_archive(
+    name = "rules_cc",
+    sha256 = "cb8ce8a25464b2a8536450971ad1b45ee309491c1f5e052a611b9e249cfdd35d",
```

Comment:
I think we don't want to use checksums now, see https://github.com/pytorch/pytorch/pull/95039

Reply:
Now the new linter becomes useful, being able to flag the line :)